src/classes/frModel.cls

```diff
@@ -120,6 +122,8 @@ public with sharing abstract class frModel {
                 + ' FROM ' + sObjectName + ' WHERE ';
             if(getSalesforceId() != null) {
                 query += 'Id = :salesforceId';
+            } else if(Test.isRunningTest()){
+                query += 'fr_Id__c = :funraiseId';
```

Comment:
I don't think this is a valid change that should be included

Reply:
This is used for testing, i can attempt to look for a workaround but may take time!

Reply:
If a test needs this to pass, then it sounds like there is a logic error elsewhere and we shouldn't patch just the test, but the problem in logic that is happening.

Reply:
Removed!